internal/driver/checkstatuses.go

```diff
@@ -19,6 +19,11 @@ import (
 // checkStatuses loops through all registered devices and tries to determine the most accurate connection state
 func (d *Driver) checkStatuses() {
 	d.lc.Debug("checkStatuses has been called")
+	start := time.Now()
+	defer func() {
+		d.lc.Debugf("checkStatuses completed in: %v", time.Since(start))
```

Comment:
Not sure if tracking this time is that helpful.

Reply:
It was useful when developing, and I think it could also be useful as a debug tool, that time could be used to inform the user the interval at which to run checkStatus for example